### PR TITLE
chore(netflix/app) convert to typescript

### PR DESCRIPTION
**Pull request by icfantv**
* convert application creation and editing controllers to typescript.

### CODE REVIEW

**Comment by icfantv** [comment]
@zanthrash @jrsquared @anotherchrisberry PTAL

File: app/scripts/modules/netflix/application/netflixCreateApplicationModal.controller.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        regionsAreIndependent: true,
+        exceptions: [],
+      };
+    }
+
+    this.application.legacyUdf = false;
+  }
+}
+export const NETFLIX_CREATE_APPLICATION_MODAL_CONTROLLER = 'spinnaker.netflix.application.create.modal.controller';
+module('spinnaker.netflix.application.create.modal.controller', [
```
Comment (anotherchrisberry) [inline app/scripts/modules/netflix/application/netflixCreateApplicationModal.controller.ts]: Should probably be `NETFLIX_CREATE_APPLICATION_MODAL_CONTROLLER`

File: app/scripts/modules/netflix/application/netflixEditApplicationModal.controller.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      $uibModalInstance: $uibModalInstance,
+      application: application,
+      applicationWriter: applicationWriter,
+      accountService: accountService,
+    }));
+  }
+}
+
+export const NETFLIX_EDIT_APPLICATION_MODAL_CONTROLLER = 'spinnaker.netflix.application.edit.modal.controller';
+module('spinnaker.netflix.application.edit.modal.controller', [
```
Comment (anotherchrisberry) [inline app/scripts/modules/netflix/application/netflixEditApplicationModal.controller.ts]: `NETFLIX_EDIT_APPLICATION_MODAL_CONTROLLER`

**Comment by anotherchrisberry** [comment]
LGTM aside from module names

**Comment by icfantv** [comment]
good catch.  fix.

**Pull request merged**